tests/foreman/ui/test_reporttemplates.py

```diff
@@ -61,10 +61,13 @@ def setup_content(module_entitlement_manifest_org, module_target_sat):
     ak = module_target_sat.api.ActivationKey(
         content_view=cv, organization=org, environment=lce, auto_attach=True
     ).create()
-    subscription = module_target_sat.api.Subscription(organization=org).search(
-        query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
-    )[0]
-    ak.add_subscriptions(data={'quantity': 1, 'subscription_id': subscription.id})
+    all_content = ak.product_content(data={'content_access_mode_all': '1'})['results']
+    for content in all_content:
+        if content['name'] == custom_repo.name:
+            content_label = content['label']
```

Comment:
I think a list-comphresions would be better here
```suggestion
     content_label = [repo['label'] for repo in all_content if repo['name'] == custom_repo.name][0]
```

Reply:
That yields a different result. One is a single element, the other is a list. So it's not a better syntax, it's a different meaning. Which one is correct?